ui.py

```diff
@@ -0,0 +1,64 @@
+import random
+from tkinter import *
+import json
+BACKGROUND_COLOR = "#B1DDC6"
+
+
+class UI:
+
+    # initializes class with starting UI
+    def __init__(self, data: list):
+        self.current_word = {}
+        self.window = Tk()
+        self.canvas = Canvas(width=800, height=526)
+        self.card_front = PhotoImage(file="images/card_front.png")
+        self.card_back = PhotoImage(file="images/card_back.png")
+        self.card_background = self.canvas.create_image(400, 263, image=self.card_front)
```

Comment:
FYI, I'm seeing some hard-coded values buried in functions.  Not worth spending a ton of time here but this wouldn't fly in my repo.